meteor/client/lib/__tests__/rundownTiming.test.ts

```diff
@@ -2304,4 +2359,183 @@ describe('rundown Timing Calculator', () => {
+		segmentsMap.set(protectString<SegmentId>(segmentId1), makeMockSegment(segmentId1, 0, rundownId))
+		segmentsMap.set(protectString<SegmentId>(segmentId2), makeMockSegment(segmentId2, 0, rundownId))
+		const parts: DBPart[] = []
+		parts.push(makeMockPart('part1', 0, rundownId, segmentId1, { expectedDuration: 1000 }))
+		parts.push(makeMockPart('part2', 0, rundownId, segmentId1, { expectedDuration: 1000 }))
+		parts.push(makeMockPart('part3', 0, rundownId, segmentId2, { expectedDuration: 1000 }))
+		parts.push(makeMockPart('part4', 0, rundownId, segmentId2, { expectedDuration: 1000 }))
+		const partInstances = convertPartsToPartInstances(parts)
+		const partInstancesMap: Map<PartId, PartInstance> = new Map()
+		const rundown = makeMockRundown(rundownId, playlist)
+		const rundowns = [rundown]
+		const result = timing.updateDurations(
+			0,
+			false,
+			playlist,
+			rundowns,
+			undefined,
+			partInstances,
+			partInstancesMap,
+			new Map(),
+			segmentsMap,
+			DEFAULT_DURATION,
+			[],
+			{
+				part2: true,
+				part3: true,
+			}
+		)
+		expect(result).toMatchObject(
+			literal<Partial<RundownTimingContext>>({
+				partsInQuickLoop: {
+					part2: true,
+					part3: true,
+				},
+			})
+		)
+	})
+})
+
+describe('findPartInstancesInQuickLoop', () => {
+	it('Returns no parts when QuickLoop is not defined', () => {
+		const { partInstances } = makeMockPartsForQuickLoopTest()
+		const playlist = makeMockPlaylist()
+
+		const result = findPartInstancesInQuickLoop(playlist, partInstances)
+
+		expect(result).toEqual({})
+	})
+
+	it('Returns parts between QuickLoop Part Markers when loop is not running', () => {
+		const { parts, partInstances } = makeMockPartsForQuickLoopTest()
+
+		const playlist = makeMockPlaylist()
+		playlist.quickLoop = {
+			start: {
+				type: QuickLoopMarkerType.PART,
+				id: parts[1]._id,
+			},
+			end: {
+				type: QuickLoopMarkerType.PART,
+				id: parts[3]._id,
+			},
+			running: false,
+			forceAutoNext: ForceQuickLoopAutoNext.DISABLED,
+			locked: false,
+		}
+
+		const result = findPartInstancesInQuickLoop(playlist, partInstances)
+
+		expect(result).toEqual({
+			[unprotectString(parts[1]._id)]: true,
+			[unprotectString(parts[2]._id)]: true,
+			[unprotectString(parts[3]._id)]: true,
+		})
+	})
+
+	it('Returns parts between QuickLoop Part Markers when loop is running', () => {
+		const { parts, partInstances } = makeMockPartsForQuickLoopTest()
+
+		const playlist = makeMockPlaylist()
+		playlist.quickLoop = {
+			start: {
+				type: QuickLoopMarkerType.PART,
+				id: parts[1]._id,
+			},
+			end: {
+				type: QuickLoopMarkerType.PART,
+				id: parts[3]._id,
+			},
+			running: false,
+			forceAutoNext: ForceQuickLoopAutoNext.DISABLED,
+			locked: false,
+		}
+
+		const result = findPartInstancesInQuickLoop(playlist, partInstances)
+
+		expect(result).toEqual({
+			[unprotectString(parts[1]._id)]: true,
+			[unprotectString(parts[2]._id)]: true,
+			[unprotectString(parts[3]._id)]: true,
+		})
+	})
```

Comment:
These two test cases look identical? Should the second one have a `running: true`?